backend/app/services/repository_job.py

```
@@ -49,36 +37,23 @@ def __init__(self, model: type[Kind] = Kind):
+            return self.load_repository_user_snapshots(db)
+
+    async def update_repositories_for_all_users(self) -> None:
         """
         Iterate through all users and update their git repositories cache
-
-        Args:
-            db_factory: Database session factory used only for loading snapshots
         """
         start_time = time.time()
         try:
             logger.info(f"[repository_job] Starting get all users task")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)